.gitlab/pipelines/stages/tests/smoke_tests.sh

```diff
@@ -34,3 +34,9 @@ wait_for_smoke_tests_pod || {
 # Follow the logs. If the tests fail, the logs command will also fail.
 pod_name="$(smoke_tests_pod_name)"
 "${KUBECTL}" logs --follow "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --container smoke-tests-smoke-tests
+
+while [ -z "$("${KUBECTL}" get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output jsonpath='{.status.containerStatuses[?(@.name == "smoke-tests-smoke-tests")].state.terminated.exitCode}')" ]; do
```

Comment:
For later refactoring: don't repeat ourselves. Something like:
```bash
jsonpath='{.status.containerStatuses[?(@.name == "smoke-tests-smoke-tests")].state.terminated.exitCode}'
while true; do
  exit_code="$("${KUBECTL}" get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output "jsonpath=${jsonpath}")"
  if [[ -n "${exit_code}" ]] ; then
    exit "${exit_code}"
  fi
  sleep 1
done
```

Reply:
Fixed in https://github.com/SUSE/kubecf/pull/115/commits/1c03df002070f9e96d11ca725f04759576ea5a5f.